tests/xrt/perf_IOPS/xcl_api_iops.cpp

```diff
@@ -40,18 +40,26 @@ load_file_to_memory(const std::string& fn)
     return bin;
 }
 
-double runTest(xclDeviceHandle handle, std::vector<std::shared_ptr<task_info>>& cmds,
-               unsigned int total)
+int
+startCmd(xclDeviceHandle handle, task_info* cmd)
+{
+  cmd->ecmd->state = ERT_CMD_STATE_NEW;
```

Comment:
KDS would set command state to ERT_CMD_STATE_NEW just before the command added to pending list.

Reply:
Yes, but that is specific to KDS and doesn't work in some flows that don't use KDS.